assembly/pom.xml

```diff
@@ -237,10 +242,13 @@
                                 db-shared,
                                 db-drda,
                                 javax.servlet-api,
-                                javax.ws.rs-api
+                                javax.ws.rs-api,
+                                metrics-core,
+                                metrics-graphite,
+                                metrics-json,
+                                metrics-jvm
```

Comment:
Why are these needed?

Reply:
Good question. These are not needed unless splice has any dependency on these versions. I wasn’t sure about it so added them in lib.

Reply:
All dependencies on metrics are transitive only.

Reply:
Ok, will remove them